### ISSUE TITLE
k-folds - create training and testing datasets

### ISSUE THREAD

**Issue by Revzik** [Revzik/Voice-Interface#58]
https://machinelearningmastery.com/k-fold-cross-validation/

Training set consists of (n-folds - 1 ) groups, test set consists of 1 group

Parameters: (n-folds lists of sound file names)
Returns: list of training data (sound file names), list of test data (sound file names)
